src/hooks/use-discount-request.js

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from 'react-query';
+import { getDiscountRequest } from 'service/company';
+
+const useDiscountRequest = (companyId) => {
+  const { data, status } = useQuery(
+    ['company-discount-request', { id: companyId }],
+    () => getDiscountRequest(companyId),
+    {
+      refetchInterval: 5000,
+      enabled: !!companyId,
+    }
+  );
+
+  const invoicesList = data?.data;
```

Comment:
Same thing as this comment: https://github.com/comarev/comarev-dashboard/pull/90#discussion_r882134226 If you feel like it makes sense.